src/js/core/widget/BaseKeyboardSupport.js

```diff
@@ -662,11 +662,11 @@
 						// corner cases, when element positions overlap
 						if (focusableElement.distance === 0) {
 							if (direction === EVENT_POSITION.down) {
-								return currentLink.compareDocumentPosition(typeof focusableElement === element ? focusableElement : focusableElement.element) &
-										Node.DOCUMENT_POSITION_CONTAINED_BY;
+								return !!(currentLink.compareDocumentPosition(typeof focusableElement === element ? focusableElement : focusableElement.element) &
```

Comment:
Do we alwas want o have boolean type?
According o func description it returns object
* @return {Object}

Reply:
This documentation refers to outer function *getNeighborhoodLinks* whereas we are changing callback function for Array.filter where true/false are expected..

Reply:
right, sorry I overlooked this